src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -150,5 +150,13 @@ public void globalLibrarySuccess() throws Exception {
 
         j.assertLogContains("[nothing here]", b);
         j.assertLogContains("map call(1,2)", b);
+        /* TODO called the wrong way: closure1([a:1])
```

Comment:
Running via a global library goes through a different code path. The symbol is looked up, then it is `call`ed. This is very different from having a definition inside the same script.